### PR TITLE
Update links to RHEL docs

### PR DESCRIPTION
**Pull request by rh-max**
#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

Fixing broken links to RHEL docs found by linkchecker.

#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [x] Foreman 3.11/Katello 4.13 (orcharhino 6.11 on EL8 only)
* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Comment by rh-max** [comment]
@maximiliankolb Hi Max, could you PTAL at the Salt repo updated links? I'm not sure every accompanying text is fitting the new link. Thanks.
https://github.com/theforeman/foreman-documentation/pull/3429/commits/641935741a67821c92dcdac4b0f1d6ddf747f385

File: guides/common/modules/ref_storage-requirements.adoc [hunk 1]
```diff
@@ -30,5 +30,5 @@ endif::[]
 ifdef::foreman-el,katello,satellite[]
 For external database servers: `{postgresql-lib-dir}` with installation size of 100 MB and runtime size of 20 GB.
 
-For detailed information on partitioning and size, see {RHELDocsBaseURL}9/html/performing_a_standard_rhel_9_installation/partitioning-reference_installing-rhel[Partitioning reference] in _Performing a standard RHEL 9 installation_.
+For detailed information on partitioning and size, see {RHELDocsBaseURL}9/html/managing_storage_devices/disk-partitions_managing-storage-devices[Disk partitions] in _Managing storage devices_.
```
Comment (Lennonka) [inline guides/common/modules/ref_storage-requirements.adoc:33]:
```suggestion
For detailed information on partitioning and size, see {RHELDocsBaseURL}9/html/managing_storage_devices/disk-partitions_managing-storage-devices[Disk partitions] in _{RHEL-abbr}{nbsp}9 Managing storage devices_.
```
Please, include the product name and version in the guide description for clarity and context.

File: guides/common/modules/con_kickstart-workflow.adoc [hunk 2]
```diff
@@ -14,4 +14,4 @@ When you run a {ProjectName} Kickstart script, the script performs the following
 . It runs user defined snippets.
 
 .Additional resources
-For more information about Kickstart, see {RHELDocsBaseURL}8/html/performing_an_advanced_rhel_8_installation/performing_an_automated_installation_using_kickstart[Performing an automated installation using Kickstart] in _Performing an advanced RHEL 8 installation_.
+For more information about Kickstart, see {RHELDocsBaseURL}8/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL_.
```
Comment (Lennonka) [inline guides/common/modules/con_kickstart-workflow.adoc:17]:
```suggestion
For more information about Kickstart, see {RHELDocsBaseURL}8/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL{nbsp}8_.
```
I think it's better to include the RHEL version in the link description.
Unlike above, the title already includes the RHEL abbreviation, so let's just add the version.
Normally, we would use the exact wording of the title, but I'd prefer to include the version number for clarity and context.

Please, also change the other occurences.

**Review by Lennonka — CHANGES_REQUESTED** [review]
Thank you, Maxi. RHEL links seem to be correct.

I recommend just a couple of minor tweaks.

File: guides/common/attributes-base.adoc [hunk 3]
```diff
@@ -187,7 +187,7 @@
 :client-content-content-view-label: AlmaLinux_9
 :client-content-product-label: {client-content-content-view-label}
 :client-content-repository-label: BaseOS
-:client-salt-minion-repository-url: https://repo.saltproject.io/py3/
+:client-salt-minion-repository-url: https://packages.broadcom.com/ui/repos/tree/General/saltproject-rpm
```
[the remark this answers is not in the document]
Comment (rh-max) [reply guides/common/attributes-base.adoc:190]: Hi @asteflova great, I didn't realize this was already in place. Removed the changes.

**Comment by rh-max** [comment]
Thanks for reviews Lena & Aneta.
I've set the cherrypicks for all supported versions, please let me know if that's incorrect.

**Comment by asteflova** [comment]
> Thanks for reviews Lena & Aneta. I've set the cherrypicks for all supported versions, please let me know if that's incorrect.

Yes, it's a good idea to try to fix the links in all branches, thanks for that. From peeking at the changes you're making, I also expect conflicts on some of those branches. Not a problem now, of course, I'm just sharing that to manage expectations :)

@rh-max Have you noticed the `deploy / build-html` check failure? Let me know if you'd like help with troubleshooting.

**Comment by asteflova** [comment]
Meanwhile, I've done a bit of a label maintenance, setting `tech review done` (because this kind of link change doesn't IMO need tech review) and `style review needed` (because there is no peer review ack yet). This is a fairly recent addition to [contributing guidelines](https://github.com/theforeman/foreman-documentation/blob/eec780f50c0a99644ae587c64c908dd2799ff936/CONTRIBUTING.md?plain=1#L15) and the exact workflow definition is still [work](https://github.com/theforeman/foreman-documentation/issues/3318) [in](https://github.com/theforeman/foreman-documentation/issues/3366) [progress](https://github.com/theforeman/foreman-documentation/issues/3155) in case you'd like to contribute with your feedback @rh-max

File: guides/common/modules/ref_kinds-of-provisioning-templates.adoc [hunk 4]
```diff
@@ -9,8 +9,8 @@ For example, a Kickstart template.
 ifndef::orcharhino[]
 For more information about Kickstart syntax and commands, see the following resources:
 
-* {RHELDocsBaseURL}9/html-single/performing_an_advanced_rhel_9_installation/index#kickstart_references[Kickstart references] in _Performing an advanced RHEL 9 installation_
-* {RHELDocsBaseURL}8/html-single/performing_an_advanced_rhel_8_installation/index#kickstart_references[Kickstart references] in _Performing an advanced RHEL 8 installation_
+* {RHELDocsBaseURL}9/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL{nbsp}9_
+* {RHELDocsBaseURL}8/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL{nbsp}8_
 * {RHELDocsBaseURL}7/html/installation_guide/sect-kickstart-syntax[Kickstart Syntax Reference] in the _{RHEL}{nbsp}7 Installation Guide_
```
Comment (rh-max) [inline guides/common/modules/ref_kinds-of-provisioning-templates.adoc:14]: Ignoring this since the offending string is part of the link.

**Comment by asteflova** [comment]
https://github.com/theforeman/foreman-documentation/pull/3411 addressed the linkchecker errors related to Salt. I'm leaving the re-review to @Lennonka.

**Comment by Lennonka** [comment]
Cherry picked:
- 2d5b1b934f..316a5b2f7e  3.12 -> 3.12
- b252fccb25..433e35631a  3.11 -> 3.11

@rh-max There are conflicts on 3.10. Please, file another PR for older branches.

**Comment by asteflova** [comment]
@Lennonka It looks like this didn't get cherry-picked into 3.13. Can you please look into it?

**Comment by Lennonka** [comment]
@asteflova Thank you for the reminder! CP'd to 3.13 and comment above updated.

**Pull request merged**